python/Ganga/new_tests/GPI/Parser/TestSelect.py

```diff
@@ -2,47 +2,68 @@
 
 from ..GangaUnitTest import GangaUnitTest
 
+job_num = 5
+job_names = ['a', 'b', 'c', 'd', 'e']
+
 class TestSelect(GangaUnitTest):
 
-    def test_SelectJob(self):
-        
-        from Ganga.GPI import Job, jobs
-        j=Job()
+    def setUp(self):
+        """Make sure that the Job object isn't destroyed between tests"""
+        super(TestSelect, self).setUp()
+        from Ganga.Utility.Config import setConfigOption
+        setConfigOption('TestingFramework', 'AutoCleanup', 'False')
```

Comment:
Actually there is some level of this going on. The objects are either SubJobXML, Registry or a plain list in some cases depending on how the RegistrySlice has been defined. I don't strictly play-with/check this in the test but I felt having it change a bit is a healthy mix to avoid problems creeping in.